### PR TITLE
Try setting a consistent background color.

### PR DESCRIPTION
**Pull request by matanlurey**
Possible (partial?) solution to https://github.com/flutter/flutter/issues/143602.

I need to do some local testing, but figured I'd let CI try it in the meantime.

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM lets try it.

File: testing/scenario_app/android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,11 +1,9 @@
 <resources>
-
-    <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+         <!-- See https://github.com/flutter/flutter/issues/143602 -->
```
Comment (jonahwilliams) [inline testing/scenario_app/android/app/src/main/res/values/styles.xml:6]: "No package ID ff found for resource ID 0xffffffff." seems like a relevant error. maybe something here isn't formatted correctly.

**Comment by reidbaker** [comment]
When reviewing goldens, here are some you might want to check for flakiness independent of this pr. 
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dcanvas_stroke_rects%26source_type%3Dflutter-engine&digest=0c68617011fdbcc1d15243fc2f5d8929
*  https://flutter-engine-gold.skia.org/detail?grouping=name%3Ddraw_vertices_texture%26source_type%3Dflutter-engine&digest=17e29c414478fc8eb094c9b5a8694d20
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dradial_gradient_clamp_transformed%26source_type%3Dflutter-engine&digest=18f8ae95b26570b53407a1fba9925f71
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dimage_shader_mirror%26source_type%3Dflutter-engine&digest=2df5a88c3f11cfc591c4588877314d3d
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dimage_shader_clamp_horiz%26source_type%3Dflutter-engine&digest=2fa4ce89de6d676fb3fe7f2990155696
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dimage_shader_horiz_mirror%26source_type%3Dflutter-engine&digest=30ae775e67c76362b04a2e12ce7103c0
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Ddraw_image_with_transform_and_offset%26source_type%3Dflutter-engine&digest=33cfea55f094c3a6fd7320cf5df9cb02
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dcanvas_stroke_joins%26source_type%3Dflutter-engine&digest=33ea6d10f8047a3dcc98c2d2eb0eeec5
* https://flutter-engine-gold.skia.org/detail?grouping=name%3Dcanvas_rrect_flipped%26source_type%3Dflutter-engine&digest=45b352e00e42d74ab63840367cc039a3
*

**Review by reidbaker — APPROVED** [review]

**Comment by matanlurey** [comment]
I wasn't able to make much progress here.

For what it's worth, I am going to monitor the current goldens and see if this problem continues, or if it was mostly due to Android 28 and OpenGLES, neither of which are CI configurations we're running at the moment.

**Pull request closed without merging**